src/ImageSharp/PixelFormats/PixelBlender{TPixel}.cs

```diff
@@ -91,7 +90,7 @@ internal abstract class PixelBlender<TPixel>
         /// <param name="source">the source span</param>
         /// <param name="amount">
         /// A span with values between 0 and 1 indicating the weight of the second source vector.
-        /// At amount = 0, "from" is returned, at amount = 1, "to" is returned.
+        /// At amount = 0, "background" is returned, at amount = 1, "source" is returned.
         /// </param>
         public void Blend<TPixelSrc>(
```

Comment:
This method has no direct tests. It shouldn't be public yet probably.

Reply:
I think it's probably ok. We've got some heavy implicit testing via the calling `Blend` method.

Reply:
The `TPixel != TPixelSrc` remains untested, but in this case the compiler should guarantee correctness in theory. ReSharper's `MemberCanBePrivate` inspections make me always suspicious though.

Reply:
I think Resharper is wrong there. We use the very same pattern with our `CloneAs<TPixel2>` APIs